### PR TITLE
[zh] Translate two SVGs in ambient/architecture

### PR DESCRIPTION
**Pull request by windsonsea**
Keep the folder structure same as en version

And translate two SVGs

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

**Comment by wilsonwu** [comment]
Another structure change is syncing, let this PR hold on a moment.

**Comment by windsonsea** [comment]
> Another structure change is syncing, let this PR hold on a moment.

All right now. Only localized two images and remove a redundant para.

File: content/zh/docs/ambient/architecture/traffic-redirection/traffic-flows-between-pods-in-ambient.svg [hunk 1]
```diff
@@ -1,21 +1,1395 @@
...
+       font-family="Helvetica, 'Segoe UI Emoji'"
+       font-size="28px"
+       fill="#1e1e1e"
+       text-anchor="middle"
+       style="white-space:pre"
+       direction="ltr"
+       dominant-baseline="text-before-edge"
+       id="text21"><tspan
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-family:SimSun;-inkscape-font-specification:SimSun"
+         id="tspan109">目的地</tspan></text>
```
Comment (wilsonwu) [inline content/zh/docs/ambient/architecture/traffic-redirection/traffic-flows-between-pods-in-ambient.svg:349]:
```suggestion
         id="tspan109">目标</tspan></text>
```

File: content/zh/docs/ambient/architecture/traffic-redirection/traffic-flows-between-pods-in-ambient.svg [hunk 2]
```diff
@@ -1,21 +1,1395 @@
...
+       font-family="Helvetica, 'Segoe UI Emoji'"
+       font-size="36px"
+       fill="#1971c2"
+       text-anchor="middle"
+       style="white-space:pre"
+       direction="ltr"
+       dominant-baseline="text-before-edge"
+       id="text15"><tspan
+   style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-family:SimSun;-inkscape-font-specification:SimSun"
+   id="tspan106">目的地工作负载</tspan>Pod</text>
```
Comment (wilsonwu) [inline content/zh/docs/ambient/architecture/traffic-redirection/traffic-flows-between-pods-in-ambient.svg:248]:
```suggestion
   id="tspan106">目标工作负载</tspan>Pod</text>
```

**Review by wilsonwu — COMMENTED** [review]
Keep the same translation in doc

**Comment by wilsonwu** [comment]
> > Another structure change is syncing, let this PR hold on a moment.
> 
> All right now. Only localized two images and removed a redundant para.

Yes, seems no conflicts, after all merged, the structure is stable.

**Review by wilsonwu — APPROVED** [review]
LGTM, thanks!

**Pull request merged**